src/format-date-range.ts

```diff
@@ -13,7 +13,12 @@ import {
   startOfQuarter,
   endOfQuarter,
   endOfYear,
+  Locale
 } from "date-fns";
+import * as Locales from 'date-fns/locale';
```

Comment:
I came up with the same solution first but then I wondered if this will affect the bundle size. 

If the date-fns library would be a peer dependency https://github.com/vercel/little-date/issues/5#issuecomment-2302439110:  What about accepting the locale object as an optional parameter instead of a locale string?  Then it will be up to the developer how the locale will be loaded (using a fixed set of locales or loaded dynamically). 

I guess this would be a breaking change.

Reply:
You're right, I initially considered accepting the locale as an object, but since we already have the option to pass the locale as a string, I wasn't sure how to approach this. Do you think it would be beneficial to accept the locale object in the options instead? However, this would mean exposing an internal dependency, the date-fns locales.